.github/labeler.yml

```diff
@@ -88,7 +88,7 @@
   - changed-files:
       - any-glob-to-any-file:
           - '**/*.md'
-          - '**/docs/**'
+          - 'docs/**'
```

Comment:
or remove it and let all .md's be picked up from

https://github.com/LavaMoat/LavaMoat/blob/716ced82813c94afdd255d7db443475959488203/.github/labeler.yml#L90

Reply:
The intent here was to pick up _any_ markdown file (except snapshots) and any present or future docs.

Reply:
agreed ^ i think `docs/**` still makes sense for our repo (like in the [examples](https://github.com/actions/labeler/blob/main/README.md))

<img width="271" alt="Screenshot 2024-01-18 at 3 51 12 pm" src="https://github.com/LavaMoat/LavaMoat/assets/1881059/bcb6861d-9bed-43e6-90ca-e64939e76f1b">

but if we plan on recursive docs folders in future (e.g. `packages/aa/docs/newDoc.md`) then i'll go ahead and close this pr

Reply:
yeah, it's just so we don't need to change it later, I guess.  tbh I don't see any harm in leaving it as-is